controllers/pagerdutyintegration/clusterdeployment_limited_support.go

```diff
@@ -39,7 +39,18 @@ func (r *PagerDutyIntegrationReconciler) handleLimitedSupport(pdclient pd.Client
 		hasLimitedSupport = val
 	}
 
+	// Check if cluster has support exception label on ClusterDeployment
+	hasSupportException := false
+	if supportExValue, err := strconv.ParseBool(cd.Labels[config.ClusterDeploymentSupportExceptionLabel]); err == nil {
```

Comment:
Should this err be handled or ignored?

Reply:
I'm not going to handle it since we should change the other too but I'd like to keep changes limited. I will validate we have labels in stage and in prod manually.

Reply:
Created https://issues.redhat.com/browse/OSD-20076